app/isomorphic/components/story-templates/blank.js

```diff
@@ -20,23 +28,29 @@ StoryCard.propTypes = {
 
 function BlankStoryTemplate(props) {
   return (
-    <div className="blank-story container">
-      <figure className="blank-story-image qt-image-16x9">
-        <ResponsiveImage
-          slug={props.story["hero-image-s3-key"]}
-          metadata={props.story["hero-image-metadata"]}
-          aspectRatio={[16, 9]}
-          defaultWidth={480}
-          widths={[250, 480, 640]}
-          imgParams={{ auto: ["format", "compress"] }}
-        />
-      </figure>
-      <h1>{props.story.headline}</h1>
-      <span className="blank-story-author">{props.story["author-name"]}</span>
-      {props.story.cards.map(card => (
-        <StoryCard key={card.id} card={card} story={props.story} />
-      ))}
-      <div className="space-before-next-story" style={{ minHeight: 100 }} />
+    <div className="container">
+      <div styleName="wrapper">
+        <WithLazy margin="20px">
+          {() => (
+            <figure className="blank-story-image" styleName="qt-image-16x9">
+              <ResponsiveImage
+                slug={props.story["hero-image-s3-key"]}
+                metadata={props.story["hero-image-metadata"]}
+                aspectRatio={[16, 9]}
+                defaultWidth={480}
+                widths={[250, 480, 640]}
+                sizes="( max-width: 120px ) 98%, ( max-width: 768px ) 48%, 23%"
+                imgParams={{ auto: ["format", "compress"], fmt: "webp" }}
+              />
+            </figure>
+          )}
+        </WithLazy>
+        <h1 styleName="headline">{props.story.headline}</h1>
+        <div styleName="author"> By {props.story["author-name"]}</div>
+        {props.story.cards.map(card => (
```

Comment:
can you add a defensive check for cards ?

Reply:
Bold will not allow you to publish a story without at least one card.